src/translations/translations_NO.json

```diff
@@ -201,10 +201,20 @@
   },
   "feedback": {
     "form_errors": "Det er feil i skjema. Disse må rettes før ressursen kan publiseres",
+    "published_warning": "Ressursen er publisert. Alle felter som endres blir lagret og publisert på nytt umiddelbart etter endring.",
     "required_field": "Påkrevd felt",
     "valid_url": "Inndata må være en gyldig URL",
     "valid_email": "Inndata må være en gyldig epost",
-    "published_warning": "Ressursen er publisert. Alle felter som endres blir lagret og publisert på nytt umiddelbart etter endring."
+    "warning_box": {
+      "contributor": "Alle felt for navn og type på bidragsyter må fylles ut, eventuelt slette unødvendige felt",
```

Comment:
"evt kan du slette unødvendig felt " ?

Reply:
Da får du en "du" i hjelpeteksten

Reply:
Endre til: "Alle felt for navn og type på bidragsyter må fylles ut, fjern eventuelt unødvendige felt"
Høres det bedre ut?